test/webdriverio/test/move_test.ts

```diff
@@ -106,34 +109,48 @@ suite('Move start tests', function () {
       // and block connected to selected block's value input.
       const info = await getFocusedNeighbourInfo(this.browser);
 
-      chai.assert(info.parentId, 'selected block has no parent block');
+      chai.assert.exists(
+        info.parentId,
+        'selected block should have parent block',
+      );
       chai.assert(
         typeof info.parentIndex === 'number',
-        'parent connection index not found',
+        'parent connection index should exist and be a number',
+      );
+      chai.assert.exists(
+        info.valueId,
+        'selected block should have child value block',
       );
-      chai.assert(info.valueId, 'selected block has no child value block');
 
       // Start move using context menu (using keyboard nav).
       await sendKeyAndWait(this.browser, [Key.Ctrl, Key.Return]);
-      await sendKeyAndWait(this.browser, 'm');
-      await keyDown(
-        this.browser,
-        (await contextMenuItems(this.browser)).findIndex(({text}) =>
-          text.includes('Move'),
-        ),
+
+      // Find how many times to press the down arrow
+      const index = (await contextMenuItems(this.browser)).findIndex(({text}) =>
+        text.includes('Move'),
+      );
+      chai.assert.isAbove(
+        index,
+        -1,
+        'expected Move to appear in context menu items',
       );
+      await keyDown(this.browser, index);
       await sendKeyAndWait(this.browser, Key.Return);
 
+      // Wait for the move icon to appear so we know we're in move mode.
+      const bubble = this.browser.$('.blocklyMoveIndicatorBubble');
+      await bubble.waitForExist();
```

Comment:
If `bubble` isn't used again then why save it?  (Unless line length?)

```suggestion
      this.browser.$('.blocklyMoveIndicatorBubble').waitForExist();
```

Reply:
Good point, I had remembered running into issues with doing this previously but it was when I needed to assert something about `bubble` which I'm not doing in this case. Fixed (had to add `await` so didn't commit your suggestion but fixed it locally and pushed)